### ISSUE TITLE
[A11y]Narrator is not reading the heading "Deprecation" on 'H' key press in scan mode 

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#7890]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1069941
Repro Steps:​
1.Start Screen Reader. Hit the URL "https://www.nuget.org" and sign in with valid credentials​
2.Tab till "Sign In" link in the top navigation and press enter​
3.Tab till "Sign in with Microsoft" button and press enter​
4.Select valid user id and press enter​
5.Enter any Username, tab till "Register" button and press enter​
6.Tab till the user menu button and press enter to expand it. Select "Manage Packages" and press enter​
7.Tab till "Published Packages", expand it and select any existing package.(If no packages are available, add a new package)​
8.Tab till "Manage Package" link under "Info" and press enter​
9.Turn on scan mode for Narrator by pressing Caps lock + Space bar
10.Now press 'H' key and check if narrator focus is navigating to headings (Deprecation) on the page​
​
Actual Result:​
Narrator is not reading the heading "Deprecation" on 'H' key press in scan mode
​
NOTE: NVDA is working as expected(added attachment for reference)

Expected Result:​
Narrator should read the headings on 'H' key press in scan mode

Notes:
I wasn't able to get scan mode working without crashing my browser. I suspect that it is simply missing an attribute that is on the other headings.

**Comment by ryuyu** [comment]
Finally got scan mode working for chrome. Unable to reproduce. Behaved as expected (each heading got a stop and read out in the same manner). closing.
